src/modules/link/graphql/mutation/link_create.rs

```diff
@@ -19,6 +19,7 @@ pub struct LinkCreate {
 #[derive(Debug, Default, InputObject)]
 pub struct LinkCreateInput {
     pub url: String,
+    #[graphql(validator(min_length = 1))]
```

Comment:
Looking great! Do you think theres a way we can use this along with the current error handling?

It would be nice to return an instance of `LinkCreate` as we do for other functions:

```rust
Ok(LinkCreate {
    link: None,
    error: Some(LinkError {
        code: LinkErrorCode::EmptyCustomHash,
        message: String::from("The custom hash cannot be empty"),
    }
})
```

Reply:
Ok, Loading the changes